### PR TITLE
Delete group join request

### PR DESCRIPTION
**Pull request by bitheadCody**

### CODE REVIEW

File: src/test/java/com/bitheads/braincloud/services/GroupServiceTest.java [hunk 1]
```diff
@@ -757,4 +791,35 @@ private String createContext(int numItemsPerPage, int startPage, String searchKe
...
     }
+
+    /**
+     * Check if the authenticated user has a join request for the provided group.
+     * 
+     * @param groupId ID of the group
+     * @return True if the authenticated user has a join request for the provided group
+     */
+    private boolean joinRequestExists(){
+        System.out.println("Checking if join request exists . . .");
```
Comment (francobithead) [inline src/test/java/com/bitheads/braincloud/services/GroupServiceTest.java]: Is the print line needed ?

File: src/test/java/com/bitheads/braincloud/services/TestFixtureBase.java [hunk 2]
```diff
@@ -21,15 +21,15 @@
...
-    static protected String m_serverUrl = "";
-    static protected String m_appId = "";
-    static protected String m_secret = "";
-    static protected String m_appVersion = "";
-    static protected String m_parentLevelName = "";
-    static protected String m_childAppId = "";
-    static protected String m_childSecret = "";
-    static protected String m_peerName = "";
-    static protected String m_redirectAppId = "";
+    static protected String m_serverUrl = "https://api.internal.braincloudservers.com/dispatcherv2";
```
Comment (francobithead) [inline src/test/java/com/bitheads/braincloud/services/TestFixtureBase.java]: I dont think we want to commit with these fields filled..

File: src/test/java/com/bitheads/braincloud/services/TestFixtureBase.java [hunk 3]
```diff
@@ -49,72 +49,84 @@ static String getServerUrl() {
     /// </summary>
     @BeforeClass
     public static void getIds() {
-        if (m_serverUrl.length() > 0)
```
Comment (francobithead) [inline src/test/java/com/bitheads/braincloud/services/TestFixtureBase.java]: I think this change will break Jenkins Java Unit testing to run. It needs to read in the ID's before testing.

**Review by nick-bithead — APPROVED** [review]

**Review by francobithead — APPROVED** [review]
LGTM

File: src/test/java/com/bitheads/braincloud/services/GroupServiceTest.java [hunk 4]
```diff
@@ -757,4 +791,34 @@ private String createContext(int numItemsPerPage, int startPage, String searchKe
...
+                }
+            }
+        } catch (JSONException je) {
+            fail("JSONException while reading GET_MY_GROUPS response");
+            je.printStackTrace();
+        }
+
+        return false;
+    }
 }
```
Comment (jo-codegirl) [inline src/test/java/com/bitheads/braincloud/services/GroupServiceTest.java:824]: I like this function.

**Pull request merged**